Makefile

```diff
@@ -39,6 +39,9 @@ libcxl.o libcxl_sysfs.o : CFLAGS += -fPIC
 libcxl.so: $(LIBNAME)
 	ln -s $(LIBNAME) libcxl.so
 
+$(LIBSONAME): $(LIBNAME)
```

Comment:
in case of fedora and openSUSE, the ldconfig is called for post-install and post-uninstall as per spec file extract below, so no reason to add above lines.
===
%post -n %{name}%{soversion} -p /sbin/ldconfig
%postun -n %{name}%{soversion} -p /sbin/ldconfig
===